scripts/es_snapshot.py

```diff
@@ -348,8 +349,8 @@ def run(self):
                 suffix = f" ({total - success} of {total} shards failed)"
             print(f"    {snapshot} [{state}]{suffix}")
             if self.detail:
-                start = datetime.fromtimestamp(snap_info['start_time']/1000)
-                end = datetime.fromtimestamp(snap_info['end_time']/1000)
+                start = datetime.fromtimestamp(snap_info['start_time']//1000)
+                end = datetime.fromtimestamp(snap_info['end_time']//1000)
```

Comment:
Do we know for sure that `snap_info['start_time']` and `snap_info['end_time']` are always ints (not floats)?

Reply:
I think you are right and it is better to use float division here. @snopoke can you confirm?

Reply:
I don't know, I'd have to check.

This script was developed for PY3 only so it shouldn't need any change.

Reply:
Good point. I'll revert changes made to this file.